README.md

```diff
@@ -75,6 +81,20 @@ $ cd cal.com
 $ yarn install
 ```
 
+Edit the turbo config file to change linter settings:
```

Comment:
I think this is only necessary if you're running `yarn build`, right? With `yarn dev` I wasn't seeing the OOM issue

Reply:
ah, yes. i can remove